python/nav/django/templatetags/info.py

```diff
@@ -164,3 +164,9 @@ def sortdict(dictionary, reverse=False):
 def is_list(value):
     """Returns True if the value is a list"""
     return isinstance(value, list)
+
+
+@register.filter
+def dunderless(value):
+    value = {k: v for k, v in value.items() if not k.startswith('__')}
+    return value
```

Comment:
I would change "value" to "mapping" to document that the value needs to behave like a dict. 

You _could_ also return the new mapping directly, though saving it like this makes for a better traceback if anything breaks, so nah. Don't do that :)